docs/autobridge.rst

```diff
@@ -0,0 +1,199 @@
+
+Floorplanning with `AutoBridge <https://github.com/Licheng-Guo/AutoBridge>`_
+----------------------------------------------------------------------------------
+
+This section covers the usage of AutoBridge in TAPA.
+
+AutoBridge is an automated floorplanning and pipelining tool for HLS designs.
+By leveraging the high-level design hierarchies,
+it produces coarse-grained floorplans so that SLR crossing do not bottleneck the
+clock frequency of an HLS kernel.
+This is natively integrated with TAPA.
+
+Supported Devices
+::::::::::::::::::::::::::::
+
+Currently AutoBridge supports the Xilinx Alveo U250 and U280 FPGAs. 
```

Comment:
I thought we only support the `xilinx_u250_xdma_201830_2` and `xilinx_u280_xdma_201920_3` platforms? Can we handle the new u250 platform now?